src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+function control() {
+  if (table.rows.length === 10) {
```

Comment:
Move 10 and 2 to variables.

Reply:
I fixed this , but i don't understand why we need this ?

Reply:
that meant to move numbers `10` and `2` to variables, e.g `const MAX_LENGTH = 10`. That will increase readability (you know what exactly is 10 and that it's the same value in all cases) and maintainable (if requirements are changed you have to change the variable value if only in one place)